_data/JEF1056/JEF1056.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Jess Fan
+institution: Dr. TJ Owens Gilroy Early College Academy
+profile_pic: JEF1056.png
+quote: I'm fortunate to be full of bad ideas. It's just unfortunate that my machine learning models have even worse ones.

```

Comment:
Looks like this quote is clocking in at 114 characters. From the boilerplate code in README.md, this can unfortunately be no more than 100 characters. If you could shorten your quote we can get you approved!

From README.md:
> name: FULLNAME-OR-NICKNAME # No longer than 28 characters
> institution: INSTITUTION-NAME 🚩 # no longer than 58 characters
> profile_pic: mona.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
> quote: YOUR-SENIOR-QUOTE # **no longer than 100 characters**, avoid using quotes(") to guarantee the format remains the same.
> github_user: YOUR-GITHUB-USERNAME

PS. Congrats on graduating from GECA! Outstanding school!